WordPressLoginFlow/src/main/res/values-v27/themes.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
     <style name="Theme.LoginFlow" parent="Base.Theme.LoginFlow">
-        <item name="android:statusBarColor">?attr/colorSurface</item>
-        <item name="android:windowLightStatusBar">true</item>
```

Comment:
I've noticed an issue with the `statusBar` color when testing WP-Android with the updates from this PR, on the login epilogue screen:

| Before | After |
| - | - |
| <img width="314" src="https://user-images.githubusercontent.com/4588074/201653884-bff6dd71-a7d1-4ee9-9ac6-781e557c07da.jpg"> | <img width="314" src="https://user-images.githubusercontent.com/4588074/201653877-fa60c962-0cd5-40de-9fd1-adc879952c44.jpg"> |

Not long ago I've encountered the same issue when working on the Jetpack Landing Screen redesign ([commit](https://github.com/wordpress-mobile/WordPress-Android/commit/df42df0407bbfbb400c6c29d74246d7af72bb533)), and by reviewing the documentation on [version specific styles](https://developer.android.com/develop/ui/views/theming/themes#Versions) I've learned that inheritance works a bit differently in styles, that in fact when we re-define a version-specific style it does not merge the properties from the main style with the properties from the version-specific one but actually overrides the whole style.

Because of the way it works, the `android:statusBarColor` and `android:windowLightStatusBar` would only be inherited in a "merging" fashion if they're declared in the `parent` style (`Base.Theme.LoginFlow`).

Though I see that the base theme only specifies colors, so maybe it's not meant to include the 2 we want to inherit 🤷 .

We could either:
1. Bring the 2 removed `statusBar` properties back to `v27` version specific style (which fixed the issue in my local test)
2. Create yet another base `parent` style from which to inherit the version-specific `Theme.LoginFlow` using the `parent` mechanism, that declares the `statusBar` properties only in the main theme file.

I hope this was clear, if not please let me know 🙇 .

Reply:
👋 @ovitrif !

> I've noticed an issue with the statusBar color when testing WP-Android with the updates from this PR, on the login epilogue screen:

Thank you so much for catching this early and before merging! 🙇 🙇 🙇 

> Not long ago I've encountered the same issue when working on the Jetpack Landing Screen redesign ([commit](https://github.com/wordpress-mobile/WordPress-Android/commit/df42df0407bbfbb400c6c29d74246d7af72bb533)), and by reviewing the documentation on [version specific styles](https://developer.android.com/develop/ui/views/theming/themes#Versions) I've learned that inheritance works a bit differently in styles, that in fact when we re-define a version-specific style it does not merge the properties from the main style with the properties from the version-specific one but actually overrides the whole style.

👍 

> Because of the way it works, the android:statusBarColor and android:windowLightStatusBar would only be inherited in a "merging" fashion if they're declared in the parent style (Base.Theme.LoginFlow).

Ahhh, this is true, I should have verifying this change myself but I assumed it would work by default, what could go wrong with style, right! I haven't worked with style long enough and next time I should be extra careful when trying to be a hero and optimize such styles instead of just letting it go... 🐈‍⬛ 

> Though I see that the base theme only specifies colors, so maybe it's not meant to include the 2 we want to inherit 🤷 .

🤷 

> We could either:
> 
> 1. Bring the 2 removed statusBar properties back to v27 version specific style (which fixed the issue in my local test)
> 2. Create yet another base parent style from which to inherit the version-specific Theme.LoginFlow using the parent mechanism, that declares the statusBar properties only in the main theme file.

I think the best and easier choice for now is for me to got with your option no.1 and revert my change there. 😊 

> I hope this was clear, if not please let me know 🙇 .

Ultra clear, once more, thank you! 🙇

Reply:
This is now done: 297fbd5340a6c89ea68ce670bd120cdbdc5c3777

Reply:
> I think the best and easier choice for now is for me to got with your option no.1 and revert my change there. 😊

Thank you for the prompt adaptation 🙇! I agree with this approach  🎉 🥇